README.md

```diff
@@ -65,6 +65,7 @@ Currently the following objects are supported,
  * [alerts](http://docs.splunk.com/Documentation/Splunk/latest/RESTREF/RESTsearch#saved.2Fsearches)
  * [dashboards](http://docs.splunk.com/Documentation/Splunk/latest/RESTREF/RESTknowledge#data.2Fui.2Fviews.2F.7Bname.7D)
  * [eventtypes](http://docs.splunk.com/Documentation/Splunk/latest/RESTREF/RESTknowledge#saved.2Feventtypes)
+ * [macros](http://docs.splunk.com/Documentation/Splunk/7.1.2/Knowledge/Usesearchmacros)
```

Comment:
This link doesn't match up with the others, as I haven't seen the admin/macros endpoint documented in the REST API reference manual.  I've only seen it mentioned in pages such as https://answers.splunk.com/answers/462327/list-of-macros.html .  I've used it successfully both to save and sync macros...